test/Tests/Strings/SequenceDistributionTests.cs

```diff
@@ -558,124 +558,6 @@ public void EnumerateSupportThrowsOnLoops()
             });
         }
 
-        [Fact]
-        public void WordModel()
```

Comment:
Why is this test deleted? Do we lose the capability to create models like the one described? From the first glance, I'd say we still could build a distribution like this using transition weights.

Reply:
That is exactly what we do.

John Guiver first implemented things this way but I convinced him that we can use transition weight in automata to achieve exactly this without new sketchy functionality in Discrete Char.

Reply:
Can we keep the test then?

Reply:
It will need to be rewritten. We have the same test in the internal project, so I didn't want to spend time on that.

Reply:
Would be nice to move that test here at some point. Doesn't necessarily have to be right now, though.

Reply:
I have returned this test dropping some things which are not supported anymore.